src/main/java/org/myrobotlab/service/config/OpenCVConfig.java

```diff
@@ -15,5 +15,9 @@ public class OpenCVConfig extends ServiceConfig {
   public boolean webViewer = false;
   public boolean capturing = false;
   public Map<String, OpenCVFilter> filters = new LinkedHashMap<>();
+  /**
+   * flip the video vertically
+   */
+  public boolean flip = false;
```

Comment:
Filters can be embedded currently, there is a filters field, the same functionality could be added by adding a Filter definition to OpenCVConfig.filters.
This is a convenience config, because of the very often occurrence or people mounting cameras upside down